Octokit.Reactive/Clients/ObservableCommitStatusClient.cs

```diff
@@ -28,10 +28,34 @@ public ObservableCommitStatusClient(IGitHubClient client)
         /// <returns></returns>
         public IObservable<CommitStatus> GetAll(string owner, string name, string reference)
         {
+            Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
+            Ensure.ArgumentNotNullOrEmptyString(name, "name");
+            Ensure.ArgumentNotNullOrEmptyString(reference, "reference");
```

Comment:
@SamTheDev here's the related discussion https://github.com/octokit/octokit.net/pull/1218#discussion_r57460248